scripts/ci/pre_commit/pre_commit_breeze_generate_autocomplete_scripts.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+import sys
+from os.path import abspath, dirname, join
+
+AIRFLOW_SOURCES_DIR = abspath(join(dirname(__file__), os.pardir, os.pardir, os.pardir))
```

Comment:
This can be done much nicer using Pathlib. See https://github.com/apache/airflow/blob/0df50f42dde4bd9b4c99cb6646416dde6fd4961e/scripts/ci/images/ci_run_docker_tests.py 

This is "newer" way of doing it - possible after we dropped Python2.